apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Snippets/BigtableClientSnippets.cs

```diff
@@ -0,0 +1,367 @@
+                Mutations.SetCell("Score", "Level 1", 12));
+
+            // Snippet: ReadRowAsync(TableName,BigtableByteString,RowFilter,CallSettings)
+            // Only read cells from the "Line 1" and "Line 2" columns in the
+            // Address column family and only take the most recent cell from
+            // each column.
+            Row row = await client.ReadRowAsync(
+                new TableName(projectId, instanceId, tableId),
+                "user12345",
+                RowFilters.Chain(
+                    RowFilters.FamilyNameRegex("Address"),
+                    RowFilters.ColumnQualifierRegex(@"Line \d"),
+                    RowFilters.CellsPerColumnLimit(1)),
+                CallSettings.FromCancellationToken(cancellationToken));
+
+            Family addressFamily = row.Families.FirstOrDefault();
+            if (addressFamily != null)
+            {
+                foreach (Column column in addressFamily.Columns)
+                {
+                    Console.WriteLine($"    Column qualifier: {column.Qualifier.ToStringUtf8()}");
+                    foreach (Cell cell in column.Cells)
+                    {
+                        Console.WriteLine($"      Cell value: {cell.Value.ToStringUtf8()}");
+                        Console.WriteLine($"      Timestamp: {cell.Version.ToDateTime()}");
+                        Console.WriteLine($"      Labels: {string.Join(", ", cell.Labels)}");
+                    }
+                }
+            }
+
+            // End snippet
+
+            foreach (Family family in row.Families)
+            {
+                Assert.Equal("Address", family.Name);
+                foreach (Column column in family.Columns)
+                {
+                    Assert.Equal(1, column.Cells.Count);
+
+                    switch (column.Qualifier.ToStringUtf8())
+                    {
+                        case "Line 1":
+                            Assert.Equal("1234 Main Street", column.Cells[0].Value.ToStringUtf8());
+                            break;
+                        case "Line 2":
+                            Assert.Equal("Apt. 2", column.Cells[0].Value.ToStringUtf8());
+                            break;
+                        default:
+                            Assert.True(false, $"Unexpected column qualifier: {column.Qualifier.ToStringUtf8()}");
+                            break;
+                    }
+                }
+            }
+        }
+
+        [Fact]
+        public async Task ReadRows()
+        {
+            var cts = new CancellationTokenSource();
+            var cancellationToken = cts.Token;
+            var client = _fixture.TableClient;
+            var projectId = _fixture.TableName.ProjectId;
+            var instanceId = _fixture.TableName.InstanceId;
+            var tableId = _fixture.TableName.TableId;
+
+            // Snippet: ReadRows(TableName,RowSet,RowFilter,long?,CallSettings)
+            // Initialize request arguments
```

Comment:
Not sure this comment line is useful.

Reply:
Yeah, this looks like it was left over from something I copy/pasted from the generated snippets.